services/core-api/app/api/projects/ams_final_application/resources/ams_final_application_resource.py

```diff
@@ -79,7 +82,27 @@ def put(self, project_summary_guid, project_summary_authorization_guid):
         final_app = final_app.update(submitter_name, documents, is_agent, pre_submitted_files, is_submitting)
         return final_app, 200
 
-    
+class AmsFinalApplicationMineSpaceEditResource(Resource, UserMixin):
```

Comment:
I don't think this functionality actually requires a separate endpoint/update method/etc, and could just be in the existing put method.

Reply:
Decided It kind off makes sense as a separate endpoint (toggling of whether it can be edited being separate from actually editing the record), so OK with this